packages/terra-doc-template/src/DocTemplate.jsx

```diff
@@ -47,6 +47,10 @@ const propTypes = {
     componentSrc: PropTypes.string,
     componentName: PropTypes.string,
   })),
+  /**
+   * Optional custom props table.
+   */
+  customPropsTable: PropTypes.node,
```

Comment:
Should we open this up to be children so people can any what they'd like?

Reply:
Updated to use children: https://github.com/cerner/terra-core/pull/1798/commits/1dcc90cddfc2dbb245b413627f8a16ef2894483c